cmd/validate/image_test.go

```diff
@@ -759,6 +759,22 @@ configuration:
 		panic(err)
 	}
 
+	testExtraRuleDataYAML := `---
+kind: ReleasePlanAdmission
+spec:
+  application: [some-app]
+  data:
+    mapping:
+	  components:
+	    - name: some-name
+		  repository: quay.io/some-namespace/msd
```

Comment:
Formatting nitpick: Probably doens't matter it looks like inconsistent indenting, maybe check the tabs/spaces inside the string.

Ps, in some places we use github.com/MakeNowJust/heredoc.Doc (sometimes as hd.Doc)  to make these kinds of strings look tidier. There's no firm policy about it, but you could use it here if you want.

Reply:
Actually for the sake of the test here, the string doesn't matter, you could set it to "bacon" if you want. :relaxed:

Reply:
..though it would matter if we wanted to unmarshal

Reply:
nice catch! I fixed the indentation on the string :)